nebula-logger/tests/log-management/classes/LogEntryHandler_Tests.cls

```diff
@@ -15,8 +15,8 @@ private class LogEntryHandler_Tests {
         ) {
             if (triggerOperationType == TriggerOperation.BEFORE_INSERT) {
                 for (LogEntry__c logEntry : (List<LogEntry__c>) triggerNew) {
-                    // The specific field changed doesn't really matter - we just want to ensure that whatever...
-                    // ...logic implement in the instance of LoggerSObjectHandlerPlugin is executed
+                    // The specific field changed doesn't really matter - we just want to ensure that whatever
+                    // logic implement in the instance of LoggerSObjectHandlerPlugin is executed
```

Comment:
implemented

Reply:
Thank you! 

On a related note, I realized a few months ago I had this habit of putting "..." at the end/beginning of wrapped lines. I guess I wanted to indicate that the sentence continued on the next line, but now when I re-read these comments, I find myself awkwardly pausing at each ellipsis. So, I now refer to this (bad) habit as my "[Christopher Walken ellipses](https://www.youtube.com/watch?v=ADFF5FrkngU)" 😆

Reply:
Haha I know _exactly_ what you mean (and the video reinforced that 😂)